source/Api.cpp

```diff
@@ -48,6 +49,7 @@ namespace Aws
             aws_s3_library_init(allocator);
             aws_event_stream_library_init(allocator);
             aws_sdkutils_library_init(allocator);
+            aws_checksums_library_init(allocator);
```

Comment:
technically we don't need to do this since aws-c-s3 will do it.

Reply:
yes, but i think its better to be explicit here and just init what you use without relying on other things doing it for you.